memcached-operator/deploy/olm-catalog/memcached-operator/0.6.0/memcached-operator.v0.6.0.clusterserviceversion.yaml

```diff
@@ -1,124 +0,0 @@
-  name: memcached-operator.v0.6.0
-  namespace: placeholder
-spec:
-  apiservicedefinitions: {}
-  customresourcedefinitions:
-    owned:
-    - kind: Memcached
-      name: memcacheds.cache.example.com
-      version: v1alpha1
-  description: Placeholder description
-  displayName: Memcached Operator
-  install:
-    spec:
-      deployments:
-      - name: memcached-operator
-        spec:
-          replicas: 1
-          selector:
-            matchLabels:
-              name: memcached-operator
-          strategy: {}
-          template:
-            metadata:
-              labels:
-                name: memcached-operator
-            spec:
-              containers:
-              - command:
-                - memcached-operator
-                env:
-                - name: WATCH_NAMESPACE
-                  valueFrom:
-                    fieldRef:
-                      fieldPath: metadata.namespace
-                - name: POD_NAME
-                  valueFrom:
-                    fieldRef:
-                      fieldPath: metadata.name
-                - name: OPERATOR_NAME
-                  value: memcached-operator
-                image: REPLACE_IMAGE
-                imagePullPolicy: Always
-                name: memcached-operator
-                resources: {}
-              serviceAccountName: memcached-operator
-      permissions:
-      - rules:
-        - apiGroups:
-          - ""
-          resources:
-          - pods
-          - services
-          - endpoints
-          - persistentvolumeclaims
-          - events
-          - configmaps
-          - secrets
-          verbs:
-          - '*'
-        - apiGroups:
-          - ""
-          resources:
-          - namespaces
-          verbs:
-          - get
-        - apiGroups:
-          - apps
-          resources:
-          - deployments
-          - daemonsets
-          - replicasets
-          - statefulsets
-          verbs:
-          - '*'
-        - apiGroups:
-          - monitoring.coreos.com
-          resources:
-          - servicemonitors
-          verbs:
-          - get
-          - create
-        - apiGroups:
-          - apps
-          resourceNames:
-          - memcached-operator
-          resources:
-          - deployments/finalizers
-          verbs:
-          - update
-        - apiGroups:
-          - cache.example.com
-          resources:
-          - '*'
-          verbs:
-          - '*'
-        serviceAccountName: memcached-operator
-    strategy: deployment
-  installModes:
-  - supported: true
-    type: OwnNamespace
-  - supported: true
-    type: SingleNamespace
-  - supported: false
-    type: MultiNamespace
-  - supported: true
-    type: AllNamespaces
-  keywords:
-  - memcached
-  - app
-  maintainers:
-  - email: corp@example.com
-    name: Some Corp
-  maturity: alpha
-  provider:
-    name: Example
-    url: www.example.com
-  replaces: memcached-operator.v0.5.0
-  version: 0.6.0
```

Comment:
Why are we removing this CSV and keeping the others?

Reply:
It was missing to be reverted. Good catch.